set.mm

```diff
@@ -49040,6 +49094,12 @@ Contrast with domain (defined in ~ df-dm ).  For alternate definitions,
     ( cxp cin wbr wcel wa cop brinxp2 df-br anbi2i 3bitr3i ) CDEABFGZHCAIDBIJZC
     DEHZJCDKZPIQSEIZJABCDELCDPMRTQCDEMNO $.
 
+  $( Intersection with a Cartesian product is a subclass of restriction.
+     (Contributed by Peter Mazsa, 19-Jul-2019.) $)
+  inxpssres $p  |- ( R i^i ( A X. B ) ) C_ ( R |` A ) $=
+    ( cxp cin cvv cres wss ssid ssv xpss12 mp2an sslin ax-mp df-res sseqtr4i )
+    CABDZEZCAFDZEZCAGQSHZRTHAAHBFHUAAIBJAABFKLQSCMNCAOP $.
```

Comment:
Nice theorem. Place it after xpss ?
(Well, actually that subsection is huge and ordering it is too difficult. That behemoth was my main target in #2861 but this will take time...)

Reply:
Right after ~xpss is an appropriate location.